packages/react/src/index.ts

```diff
@@ -8,6 +8,7 @@ export * from './checkout';
 export * from './helpers';
 export * from './products';
 export * from './locale';
+export * from './payment';
```

Comment:
We are using `payments` folder in `redux` and `client` packages. Can you change it to be `payments` as well, please?